### PR TITLE
Red wash

### PR DESCRIPTION
**Pull request by huangjien**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# [Choice] Debian OS version (use bullseye on local arm64/Apple Silicon): bookworm, buster, bullseye
+ARG VARIANT="bullseye"
+# FROM --platform=linux/amd64 mcr.microsoft.com/devcontainers/base:${VARIANT}
+FROM mcr.microsoft.com/devcontainers/base:${VARIANT}
+
+# [Optional] Uncomment this section to install additional OS packages.
+# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+#     && apt-get -y install --no-install-recommends <your-package-list-here>
```
Comment (ourboy) [inline .devcontainer/Dockerfile:8]: add a new line

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	"overrideFeatureInstallOrder": [
+	  "ghcr.io/devcontainers/features/common-utils",
+	  "ghcr.io/devcontainers/features/git",
+	  "ghcr.io/devcontainers/features/github-cli:1",
+	  "ghcr.io/devcontainers-contrib/features/curl-apt-get:1",
+	  "ghcr.io/devcontainers-contrib/features/zsh-plugins",
+	  "ghcr.io/devcontainers/features/docker-in-docker"
+	  //"ghcr.io/jungaretti/features/make:1"
+	]
+  }
```
Comment (ourboy) [inline .devcontainer/devcontainer.json:29]: add a new line

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Deploy the collection
+      uses: artis3n/ansible_galaxy_collection@v2
+      with:
+        api_key: ${{ secrets.GALAXY_TOKEN }}
```
Comment (ourboy) [inline .github/workflows/release.yml:21]: add a new line

File: galaxy.yml [hunk 4]
```diff
@@ -4,13 +4,13 @@
...
 # underscores or numbers and cannot contain consecutive underscores
-namespace: ibm
+namespace: merative
 
 # The name of the collection. Has the same character restrictions as 'namespace'
 name: spm_toolbox
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.0.1
+version: 1.0.2
```
Comment (ourboy) [inline galaxy.yml:13]: I would move this to version 1.1.0 or 2.0.0, clean break

**Review by ourboy — CHANGES_REQUESTED** [review]

**Review by ourboy — APPROVED** [review]

**Review by martinccummins — APPROVED** [review]
LGTM

**Pull request merged**